wgpu-types/src/lib.rs

```diff
@@ -3533,6 +3533,10 @@ impl TextureFormat {
 
         flags.set(TextureFormatFeatureFlags::FILTERABLE, is_filterable);
         flags.set(TextureFormatFeatureFlags::BLENDABLE, is_blendable);
+        flags.set(
+            TextureFormatFeatureFlags::STORAGE_WRITE_ONLY,
+            allowed_usages.contains(TextureUsages::STORAGE_BINDING),
+        );
```

Comment:
This isn't quite right, looks like my earlier comment was misleading. According to [this table](https://www.w3.org/TR/webgpu/#texture-format-caps) `bgra8Unorm` textures have the `STORAGE_WRITE_ONLY` capability if the "bgra8unorm-storage" feature has been requested.

So the condition for setting this flag should be `*self == Self::Bgra8Unorm && device_features.contains(Features::BGRA8UNORM_STORAGE)
`